test/unit/models/flaggedItem/notifyUtils.test.js

```diff
@@ -67,7 +67,7 @@ describe('sendToCommunities', () => {
     var expectedUserIds = [modIds1, modIds2]
     var expectedText = [`${message} 1`, `${message} 2`]
 
-    const hyloAdminIds = process.env.HYLO_ADMINS && process.env.HYLO_ADMINS.split(',').map(id => Number(id))
+    const hyloAdminIds = process.env.HYLO_ADMINS.split(',').map(id => Number(id))
```

Comment:
this will fail on any environment that doesn't have `HYLO_ADMINS` set including a local dev env. better to explicitly define the value to what you want for the purposes of the test rather than imposing required configuration on all test environments.

Reply:
Then you can also get rid of the `if` wrapper below, so the test then becomes fully deterministic.